sdk/cosmosdb/cosmos/src/client/Container/ContainerRequest.ts

```diff
@@ -7,5 +7,11 @@ import { VerboseOmit } from "../../utils/types";
 export interface ContainerRequest extends VerboseOmit<ContainerDefinition, "partitionKey"> {
```

Comment:
This needs to be exported from the `index.ts`. Looks like it isn't now which is my fault: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/cosmosdb/cosmos/src/index.ts